### PR TITLE
Template inclusions

### PR DESCRIPTION
**Pull request by OneCricketeer**
Consolidate Common Property settings into `templates/includes` for each role.

### CODE REVIEW

File: roles/confluent.connect-distributed/templates/includes/base_connect-distributed.properties.j2 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+bootstrap.servers={% for host in groups['broker'] %}{% if loop.index > 1%},{% endif %}{{ host }}:{{broker.config.port}}{% endfor %}
+
+value.converter.schema.registry.url={% for host in groups['schema-registry'] %}{% if loop.index > 1%},{% endif %}http://{{ host }}:8081/{% endfor %}
+
+key.converter.schema.registry.url={% for host in groups['schema-registry'] %}{% if loop.index > 1%},{% endif %}http://{{ host }}:8081/{% endfor %}
```
[the remark this answers is not in the document]
Comment (cotedm) [reply roles/confluent.connect-distributed/templates/includes/base_connect-distributed.properties.j2]: yes, that's correct #12 is "to do" and certs need to be better to really do this effectively.

**Comment by cotedm** [comment]
@cricket007 high level looks nice, have you tried it out yourself yet? I think that's the safest course here because reviewing is looking for typos essentially. I can give it a test if you haven't done it yet.

**Comment by cricket007** [comment]
I tested out only a broker so far, but I can do a more thorough test now that my Vagrant setup is working as expected

**Comment by cricket007** [comment]
@cotedm All working as expected   ✅

**Pull request merged**